py/nupic/examples/mnist.py

```diff
@@ -63,8 +63,8 @@ def load_images(file_name):
                 # vec = [ord(c) for c in vec]   # python2
                 vec = list(vec)
                 vec = np.array(vec, dtype=np.uint8)
-                buf = np.reshape(vec, (rows, cols, 1))
-                imgs.append(buf)
+                # vec = np.reshape(vec, (rows, cols, 1))
```

Comment:
why is the reshape commented out now?

Reply:
Ok, I reverted a few changes.  The input images have dimensions again.